src/Cell.tsx

```diff
@@ -34,15 +34,15 @@ function Cell<R, SR>({
   row,
   rowIdx,
   dragHandle,
-  skipCellFocusRef,
+  shouldFocusCellRef,
```

Comment:
Now we only focus if `shouldFocusCellRef.current` is `true`. Wonder if this should be a state instead? But then resetting it would trigger a re-render